apps/platform-app/src/app/[locale]/(auth)/(components)/LinkedInLoginButton/LinkedInLoginButton.tsx

```diff
@@ -0,0 +1,32 @@
+'use client'
+import { AppRoutes } from '@/utils/routes'
+import { Button } from '@gdh/ui-system'
+import { GithubIcon } from '@gdh/ui-system/icons'
```

Comment:
**issue**: The component is using `GithubIcon` for LinkedIn login. Consider replacing it with a LinkedIn icon for consistency.